vm/chef/cookbooks/singlefs/files/singlefs-monitoring

```diff
@@ -0,0 +1,426 @@
+#!/bin/bash -eu
```

Comment:
This script is extremely long and does not offer any modularization - could we split it into logical functions and call them in a more descriptive way?